Mathlib/Topology/ExtremallyDisconnected.lean

```diff
@@ -140,12 +138,15 @@ end
 
 section
 
-variable {A B C D : Type u} {E : Set D}
-  [TopologicalSpace A] [TopologicalSpace B] [TopologicalSpace C] [TopologicalSpace D]
+variable {A B C D E : Type u} [TopologicalSpace A] [TopologicalSpace B] [TopologicalSpace C]
+  [TopologicalSpace D] [TopologicalSpace E]
```

Comment:
Whoops, thank you!

Reply:
I think I want to keep both `D` and `E` for now, so that the resulting "Zorn subset" variables in `exists_compact_surjective_zorn_subset` is consistent with the variables used in the later lemmas.